assignment/scripts/3-music-collection.js

```diff
@@ -1 +1,67 @@
 console.log('***** Music Collection *****')
+let collection = [];
+console.log(collection);
+
+function addToCollection(title, artist, yearPublished) {
+let records = {
```

Comment:
Beware indentation. It won't make a difference to how your code runs, but it can make a big difference for developers reading your code later. The typical pattern is to either open/close brackets on the same line if it is short, OR:

1. Put an opening bracket at the end of one line
2. Indent all subsequent lines that are contained within the brackets
3. Put the closing bracket at the same level of indentation as the line with the opening bracket

```js
function addToCollection(title, artist, yearPublished) {
  let records = {
    title: title,
    artist: artist,
    yearPublished: yearPublished
  };

  collection.push(records);
  return records;
}
```

This helps makes it easier to tell at a glance which code is located within what brackets.